ngraph_bridge/ops/ngraph_ops.cc

```diff
@@ -1,34 +0,0 @@
-/*******************************************************************************
```

Comment:
This is a TF convention to keep Op registrations in a separate file from the kernel. Intel-TF also follows the same and I think we should keep this file even though there is only 1 op that we register.

Reply:
Ok, I'll do the refactoring to use canonical TF convention in a follow-up PR.

Reply:
See #710